packages/frontend/core/src/modules/doc-display-meta/services/doc-display-meta.ts

```diff
@@ -52,6 +53,7 @@ interface DocDisplayIconOptions<T extends IconType> {
 }
 interface DocDisplayTitleOptions {
   originalTitle?: string;
+  title?: string; // title alias
```

Comment:
DocDisplayService no need to know alias.

alias should be handled at block level